README
This is a collection of progams for 2D and 3D beam and truss structures.

WARNING: before using one of these programs make sure you 
         backup your previous work ! and make sure that 
         you apply these in a safe sandbox (not having 
         an influence on other work).

There happens to be some documentation included in both the
source (files) and in the doc directory, the latter if this
was available at the time of the publication.

Kind regards, have fun and the usual YMMV disclaimers apply.


ChangeLog
commit 0496dfe44117b03ef036886145836e37eeacaee2
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:37:29 2014 +0100

    Added LICENSE file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit dbdbc64721912fc990c7ab47ccc89fd5717c56a7
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:37:11 2014 +0100

    Added TODO file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit efbd437a9098ce5cf934aa6341320ba0668f4f85
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:36:54 2014 +0100

    Added NEWS file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit da78f3cdb8a66e6470238e454741bbd3c6c0b99c
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:36:41 2014 +0100

    Added ISSUES file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit 065a44cddae220dd92272363165e4b2ff47dd384
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:35:34 2014 +0100

    Added INSTALL file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit 55289988dec901a46b5b4531ebc6db9aa149eb97
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:35:02 2014 +0100

    Added COPYING file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit 55220d929c7a1ecac1335023db465ba857dc72f6
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:34:46 2014 +0100

    Added CONTRIBUTING file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit a7d6494f7dcd36bfad41e1807ac2bc229017c2d7
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:21:40 2014 +0100

    Added BUGS file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit d7221e69d4e46a904f705b8016040899dcca24f6
Author: bert <bert.timmerman@xs4all.nl>
Date:   Sun Dec 21 14:21:21 2014 +0100

    Added AUTHORS file.
    
    Signed-off-by: bert <bert.timmerman@xs4all.nl>

commit a8ab40cb6bac307b4b50c208141b7efe63de011b
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Fri Apr 27 08:23:17 2012 +0200

    NEN6772.c: added F_g_t_u_d().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit 5b672f13bd12aad90525167019c1f5edfe56e803
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Fri Apr 27 00:15:41 2012 +0200

    NEN6772.c: minor corrections.
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit 43d843d40c91ca99019934aff798a750d56d7ae7
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Fri Apr 27 00:08:06 2012 +0200

    NEN6772.c: added F_p_d().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit ab1c251065cbbfd962c32a1ae4c5d93db8a1d93d
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Wed Apr 18 21:12:57 2012 +0200

    NEN6772.c: minor corrections.
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit f069a98b6867514f7c348555b8f067065c0eef16
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Wed Apr 18 21:10:51 2012 +0200

    NEN6772.c: added F_g_u_d().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit dffa3113bec4e913c089536bef6c2a077e5d4866
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Wed Apr 18 16:21:19 2012 +0200

    NEN6772.c: added UC_11_1_12().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit f46923612f9e8a1b327c89cc2eed5cb58c0c4be6
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Wed Apr 18 16:17:40 2012 +0200

    NEN6772.c: added UC_11_1_11().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit e8981c47e8c2c5e9d0bd5cd6f738f2b4e88a4479
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Mon Apr 16 21:31:56 2012 +0200

    Ignore stuff in doc/.
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit e0f180e1ce9df72638f562aece16c2f3d5bcba04
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Tue Apr 10 21:58:18 2012 +0200

    NEN6772.c: bumped up the copyright year.
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit bc28a1d872d3c12e87a8bf71968cd0638b4516d4
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Tue Apr 10 21:55:39 2012 +0200

    NEN6772.c: added UC_11_1_10().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit a62a6cdef83b2d692715648997699cf9a854d84c
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Tue Apr 10 21:50:22 2012 +0200

    NEN6772.c: added t_ef().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit 85664abcfe8a0c099ce24da3a4363995ec1c39e2
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Tue Apr 10 20:27:34 2012 +0200

    NEN6772.c: added UC_11_1_8().
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit c441fc7e9897666a94d8ffc58dc8a97cb784213e
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Tue Apr 10 20:15:25 2012 +0200

    NEN6772.c: added a version (year of release) of the standard.
    
    Removed trailing white space.
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit 4897b463511f404ba1625f74d4a9e7389706e0e3
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Tue Apr 10 07:37:43 2012 +0200

    Updated the Makefile.
    
    Signed-off-by: Bert Timmerman <bert.timmerman@xs4all.nl>

commit d085bf98b2fcdda96db9007374fa98c7636a5656
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Sat Apr 30 23:33:29 2011 +0200

    NEN6772: added a calculation of the reduction factor for a rivet connection.

commit b610b3856721286ee6c47c756b79c157b9874638
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Fri Apr 29 00:01:55 2011 +0200

    NEN6772: added a calculation of the swaging factor for a rivet connection.

commit 4a8dafe3e87fc91c56687959f7af8838f3cb4ee5
Author: Bert Timmerman <bert.timmerman@xs4all.nl>
Date:   Thu Apr 28 23:05:16 2011 +0200

    NEN6772: added a calculation of the ultimate tension capacity for a rivet.

commit e75b264d479b77dcafc414f5e4669e356cdb429c

src/test.c
/*!
 * \file test.c
 *
 * \brief Test program.
 *
 * <b>Author:</b>\n
 * Copyright (C) 2008, 2011 by Bert Timmerman <bert.timmerman@xs4all.nl>
 * 
 * <b>License:</b>\n
 * This program is free software; you can redistribute it and/or modify\n
 * it under the terms of the GNU General Public License as published by\n
 * the Free Software Foundation; either version 2 of the License, or\n
 * (at your option) any later version.\n
 * \n
 * This program is distributed in the hope that it will be useful,\n
 * but WITHOUT ANY WARRANTY; without even the implied warranty of\n
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 * See the GNU General Public License for more details.\n
 * \n
 * You should have received a copy of the GNU General Public License\n
 * along with this program; if not, write to:\n
 * the Free Software Foundation, Inc.,\n
 * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.\n
 */


#include <stdio.h>
#include <string.h>
#include <stdlib.h>


int
main (int argc, char *argv[])
{
        FILE *F_IR3;
        char InputFileName[] = "../test/03093-01.IR3";
        int length = 1024;
        char *TempText;
        char *Token;
        char Delimiters[] = " ()<>*;:-";
        ssize_t ErrorCode;

        F_IR3 = fopen (InputFileName, "r");
        if (F_IR3 != NULL)
        {
                fprintf (stderr, "Input file %s is opened Read-Only.\n", InputFileName);
        }
        else
        {
                fprintf (stderr, "Error: input file %s not found or does not exist.\n", InputFileName);
                exit (EXIT_FAILURE);
        }
        while (!feof (F_IR3))
        {
                TempText = (char *) malloc (length + 1);
                ErrorCode = getline (&TempText, &length, F_IR3);
                if (ErrorCode != -1)
                {
                        fprintf (stderr, "%s", TempText);
                        fprintf (stderr, "Start splitting into tokens.\n");
                        Token = strtok(TempText, Delimiters);
                        while (Token != NULL)
                        {
                                fprintf (stderr, "%s\n", Token);
                                Token = strtok (NULL, " ");
                        }
                        fprintf (stderr, "Ready splitting into tokens.\n");
                }
                else
                {
                        if (ErrorCode == EOF)
                        {
                                fprintf (stderr, "End Of File encountered.\nDone.\n");
                                exit (EXIT_FAILURE);
                        }
                        else
                        {
                                fprintf (stderr, "Error while reading line from file %s.\n", InputFileName);
                                exit (EXIT_FAILURE);
                        }
                }
        }
        ErrorCode = fclose (F_IR3);
        if (!ErrorCode)
        {
                fprintf (stderr, "Input file %s is closed.\n", InputFileName);
        }
        else
        {
                fprintf (stderr, "Error: input file %s not closed or does not exist.\n", InputFileName);
                exit (EXIT_FAILURE);
        }
        return (EXIT_SUCCESS);
}


/* EOF */



src/i_prof.c
/*!
 * \file i_prof.c
 *
 * \brief This unit generates the properties of (prismatic) I-shaped profiles.
 *
 * \author Copyright 2014 Bert Timmerman <bert.timmerman@xs4all.nl>
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.\n
 * \n
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU General Public License for more details.\n
 * \n
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 * MA 02110-1301, USA.\n
 *
 * This unit generates the properties of (prismatic) I-shaped profiles.\n
 * \n
 * All standard available profiles are included (whatever that means ;-)\n
 * \n
 * \warning ALL x-, y- and z-indices refer to the LOCAL coordinate
 * system of the profile.\n
 *
 * \warning All units in SI-metrics [m, kg, sec] or derivatives, and are
 * stated below between [].\n
 * \n
 *
 * Include the generation of properties of I-shaped profiles based
 * on input with parameters.\n
 * For instance parameter input in the form of the ProfileName = 
 * 'I500x300x14.5x28x27x0'.\n
 * Meaning:\n
 * I = I-shaped profile
 * 500 = height
 * x = delimiter (typical)
 * 300 = width
 * 14.5  = web thickness
 * 28 = flange thickness
 * 27 = radius between web and flanges
 * 0 = radius on outer edge of the flanges
 * Thus the properties of welded (double) symmetrical I-shaped profiles
 * can be generated.\n
 */


/* EOF */


src/ipe_prof.c
/*!
 * \file ipe_prof.c
 *
 * \brief This unit generates the properties of (prismatic) I-shaped
 * profiles (IPE class).
 *
 * \author Copyright 2014 Bert Timmerman <bert.timmerman@xs4all.nl>
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.\n
 * \n
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU General Public License for more details.\n
 * \n
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 * MA 02110-1301, USA.\n
 *
 * This unit generates the properties of (prismatic) I-shaped profiles.\n
 * \n
 * All standard available profiles are included (whatever that means ;-)\n
 * \n
 * \warning ALL x-, y- and z-indices refer to the LOCAL coordinate
 * system of the profile.\n
 *
 * \warning All units in SI-metrics [m, kg, sec] or derivatives, and are
 * stated below between [].\n
 * \n
 */


/*!
 * Description of fields in the table (with units between []):\n
 * <ul>
 * <li> ProfileName [-]</li>
 * <li> h [mm]</li>
 * <li> b [mm]</li>
 * <li> s [mm]</li>
 * <li> t [mm]</li>
 * <li> r [mm]</li>
 * <li> r1 [mm]</li>
 * <li> A [cm2]</li>
 * <li> c [mm]</li>
 * <li> h1 [mm]</li>
 * <li> wmin [mm]</li>
 * <li> wmax [mm]</li>
 * <li> d [mm]</li>
 * <li> Ixx [cm4]</li>
 * <li> Iyy [cm4]</li>
 * <li> iy [mm]</li>
 * <li> Wyy [cm3]</li>
 * <li> Syy [cm3]</li>
 * <li> Izz [cm4]</li>
 * <li> iz [mm]</li>
 * <li> Wzz [cm3]</li>
 * <li> m [kg/m]</li>
 * <li> ct [-]</li>
 * <li> </ul>
 */
enum ipe_table_headings
(
        I_name,
        I_h,
        I_b,
        I_s,
        I_t,
        I_r,
        I_r1,
        I_A,
        I_c,
        I_h1,
        I_wmin,
        I_max,
        I_d,
        I_Ixx,
        I_Iyy,
        I_iy,
        I_Wyy,
        I_Syy,
        I_Izz,
        I_iz,
        I_Wzz,
        I_m,
        I_ct,
        I_NUMCOLS
)

#define I_NUMROWS 1024

char ipe_prof[I_NUMROWS][I_NUMCOLS]=
{
"IPE80","80","46","3.8","5.2","5","0","764","10","60","0","0","0","0.6727","80.1","32.4","20","11.6","8.49","10.5","3.69","6","#ct"
"IPE100","100","55","4.1","5.7","7","0","1030","12.5","75","0","0","0","1.153","171","40.7","34.2","19.7","15.9","12.4","5.79","8.1","#ct"
"IPE120","120","64","4.4","6.3","7","0","1320","13.5","93","0","0","0","1.689","318","49","53","30.4","27.7","14.5","8.65","10.4","#ct"
"IPE140","140","73","4.7","6.9","7","0","1640","14","112","38","38","10","2.401","541","57.4","77.3","44.2","44.9","16.5","12.3","12.9","#ct"
"IPE160","160","82","5","7.4","9","0","20.1","16.5","127","42","42","12","3.530","869","65.8","109","61.9","68.3","18.4","16.7","15.8","#ct"
"IPE180","180","91","5.3","8","9","0","23.9","17","146","48","48","12","4.723","1317","74.2","146","83.2","101","20.5","22.2","18.8","#ct"
"IPE200","200","100","5.6","8.5","12","0","28.5","20.5","159","54","54","12","6.846","1943","82.6","194","110","142","22.4","28.5","22.4","#ct"
"IPE220","220","110","5.9","9.2","12","0","33.4","21","178","58","58","16","8.982","2772","91.1","252","143","205","24.8","37.3","26.2","#ct"
"IPE240","240","120","6.2","9.8","15","0","39.1","25","190","64","64","16","12.74","3892","99.7","324","183","284","26.9","47.3","30.7","#ct"
"IPE270","270","135","6.6","10.2","15","0","45.9","25","220","72","72","16","15.71","5790","112","429","242","420","30.2","62.2","36.1","#ct"
"IPE300","300","150","7.1","10.7","15","0","53.8","25.5","249","80","80","20","19.75","8356","125","557","314","604","33.5","80.5","42.2","#ct"
"IPE330","330","160","7.5","11.5","18","0","62.6","29.5","271","86","86","20","27.59","11770","137","713","402","788","35.5","98.5","49.1","#ct"
"IPE360","360","170","8","12.7","18","0","72.7","30.5","299","92","92","22","37.08","16270","150","904","510","1043","37.9","123","57.1","#ct"
"IPE400","400","180","8.6","13.5","21","0","84.5","34.5","331","96","96","22","50.41","23130","165","1160","654","1318","39.5","146","66.3","#ct"
"IPE450","450","190","9.4","14.6","21","0","98.8","35.5","379","102","102","24","66.05","33740","185","1500","851","1676","41.2","176","77.6","#ct"
"IPE500","500","200","10.2","16","21","0","116","37","426","108","108","24","88.62","48200","204","1930","1100","2142","43.1","214","90.7","#ct"
"IPE550","550","210","11.1","17.2","24","0","134","41","468","116","116","24","121.7","67120","223","2440","1390","2668","44.5","254","106","#ct"
"IPE600","600","220","12","19","24","0","156","43","514","120","120","24","164.6","92080","243","3070","1760","3387","46.6","308","122","#ct"
}


/* EOF */


src/inp_prof.c
/*!
 * \file inp_prof.c
 *
 * \brief This unit generates the properties of (prismatic) I-shaped
 * profiles (INP class).
 *
 * \author Copyright 2014 Bert Timmerman <bert.timmerman@xs4all.nl>
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.\n
 * \n
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU General Public License for more details.\n
 * \n
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 * MA 02110-1301, USA.\n
 *
 * This unit generates the properties of (prismatic) I-shaped profiles.\n
 * \n
 * All standard available profiles are included (whatever that means ;-)\n
 * \n
 * \warning ALL x-, y- and z-indices refer to the LOCAL coordinate
 * system of the profile.\n
 *
 * \warning All units in SI-metrics [m, kg, sec] or derivatives, and are
 * stated below between [].\n
 * \n
 */


/*!
 * Description of fields in the table (with units between []):\n
 * <ul>
 * <li> ProfileName [-]</li>
 * <li> h [mm]</li>
 * <li> b [mm]</li>
 * <li> s [mm]</li>
 * <li> t [mm]</li>
 * <li> r [mm]</li>
 * <li> r1 [mm]</li>
 * <li> A [cm2]</li>
 * <li> c [mm]</li>
 * <li> h1 [mm]</li>
 * <li> wmin [mm]</li>
 * <li> wmax [mm]</li>
 * <li> d [mm]</li>
 * <li> Ixx [cm4]</li>
 * <li> Iyy [cm4]</li>
 * <li> iy [mm]</li>
 * <li> Wyy [cm3]</li>
 * <li> Syy [cm3]</li>
 * <li> Izz [cm4]</li>
 * <li> iz [mm]</li>
 * <li> Wzz [cm3]</li>
 * <li> m [kg/m]</li>
 * <li> ct [-]</li>
 * <li> </ul>
 */
enum inp_table_headings
(
        I_name,
        I_h,
        I_b,
        I_s,
        I_t,
        I_r,
        I_r1,
        I_A,
        I_c,
        I_h1,
        I_wmin,
        I_max,
        I_d,
        I_Ixx,
        I_Iyy,
        I_iy,
        I_Wyy,
        I_Syy,
        I_Izz,
        I_iz,
        I_Wzz,
        I_m,
        I_ct,
        I_NUMCOLS
)

#define I_NUMROWS 1024

char inp_prof[I_NUMROWS][I_NUMCOLS]=
{
"INP80","80","42","3.9","5.9","3.9","2.3","760","10.5","59","0","0","0","#Ixx","77.8","32","19.5","11.4","6.29","9.1","3","5.95","#ct"
"INP100","100","50","4.5","6.8","4.5","2.7","1060","12.5","75","0","0","0","#Ixx","171","40.1","34.2","19.9","12.2","10.7","4.88","8.32","#ct"
"INP120","120","58","5.1","7.7","5.1","3.1","1420","14","92","0","0","0","#Ixx","328","48.1","54.7","31.8","21.5","12.3","7.41","11.2","#ct"
"INP140","140","66","5.7","8.6","5.7","3.4","1830","15.5","109","34","34","10","#Ixx","573","56.1","81.9","47.7","35.2","14","10.7","14.4","#ct"
"INP160","160","74","6.3","9.5","6.3","3.8","2280","17.5","125","38","38","12","#Ixx","935","64","117","68","54.7","15.5","14.8","17.9","#ct"
"INP180","180","82","6.9","10.4","6.9","41","2790","19","142","44","44","12","#Ixx","1450","72","161","93.4","81.3","17.1","19.8","21.9","#ct"
"INP200","200","90","7.5","11.3","7.5","4.5","3350","20.5","159","46","46","16","#Ixx","2140","80","214","125","117","18.7","26","26.3","#ct"
"INP220","220","98","8.1","12.2","8.1","4.9","3960","22.5","175","52","52","16","#Ixx","3060","88","278","162","162","162","20.2","31.1","#ct"
"INP240","240","106","8.7","13.1","8.7","5.2","4610","24","192","56","56","16","#Ixx","4250","95.9","354","206","221","22","41.7","36.2","#ct"
"INP260","260","113","9.4","14.1","9.4","5.6","5340","26","208","58","58","20","#Ixx","5740","104","442","257","288","23.2","51","41.9","#ct"
"INP280","280","119","10.1","15.2","10.1","6.1","6110","27.5","225","62","62","20","#Ixx","7590","111","542","316","364","24.5","61.2","48","#ct"
"INP300","300","125","10.8","16.2","10.8","6.5","6910","29.5","241","64","64","20","#Ixx","9800","119","653","381","451","25.6","72.2","54.2","#ct"
"INP320","320","131","11.5","17.3","11.5","6.9","7780","31.5","257","70","70","20","#Ixx","12501","127","782","457","555","26.7","84.7","61.1","#ct"
"INP340","340","137","12.2","18.3","12.2","7.3","8680","33","274","74","74","20","#Ixx","15700","135","923","540","674","28","98.4","68.1","#ct"
"INP360","360","143","13","19.5","13","7.8","9710","35","290","74","74","22","#Ixx","19610","142","1090","638","818","29","114","76.2","#ct"
"INP380","380","149","13.7","20.5","13.7","8.2","10700","37","306","80","80","22","#Ixx","24010","150","1260","741","975","30.2","131","84","#ct"
"INP400","400","155","14.4","21.6","14.4","8.6","11800","38.5","323","84","84","22","#Ixx","29210","157","1460","857","1160","31.3","149","92.6","#ct"
"INP425","425","163","15.3","23","15.3","9.2","13200","41","343","86","86","24","#Ixx","36970","167","1740","1020","1440","33","176","104","#ct"
"INP450","450","170","16.2","24.3","16.2","9.7","14700","43.5","363","92","92","24","#Ixx","45850","177","2040","1200","1730","34.3","203","115","#ct"
"INP475","475","178","17.1","25.6","17.1","10.3","16300","45.5","384","96","96","24","#Ixx","56480","186","2380","1400","2090","36","235","128","#ct"
"INP500","500","185","18","27","18","10.8","18000","48","404","100","100","24","#Ixx","68740","196","2750","1620","2480","37.2","268","141","#ct"
"INP550","550","200","19","30","19","11.9","21300","53","444","110","110","24","#Ixx","99180","216","3610","2120","3490","40.2","349","167","#ct"
"INP600","600","215","21.6","32.4","21.6","13","25400","57.5","485","120","120","24","#Ixx","139000","234","4630","2730","4670","43","434","199","#ct"
}


/* EOF */
